packages/components/src/stories/Introduction.mdx

```diff
@@ -53,12 +54,19 @@ or selectively:
 import '@plone/components/src/styles/basic/TextField.css';
 ```
 
+### CSS layers
+
+This package use CSS layers to provide a way to style the components in a more flexible way.
+It uses the `plone-components` layer name to scope all the CSS declarations in the package.
+The basic styling uses the nested `plone-components.base` named layer.
+You can use the `plone-components` layer to override the basic styling, or use the `plone-components.base` layer to override the basic styling in a more specific way.
 
 ## Quanta
```

Comment:
I think this needs to increase a level because it is under Styling, not a sibling to it.

```suggestion
### Quanta
```